corecito_account.py

```diff
@@ -76,16 +76,16 @@ async def get_balances(self):
             'core_number_currency_balance': core_number_currency_balance,
             'core_number_currency_available': core_number_currency_available})
 
-  async def order_market_buy(self, quantity=0.0):
+  async def order_market_buy(self, tx_result, quantity=0.0):

```

Comment:
Because `tx_result` and `quantity` are in different currencies (BTC & ETH), they must use a different max amount of decimals (6 & 3)

Reply:
Do you think it would be a good idea to have max amount of decimal for buy/sell parameters for each exchange in the config? I mean, something like:

`cryptocom_max_decimals_buy: 6
cryptocom_max_decimals_sell: 6
binance_max_decimals_buy: 3
binance_max_decimals_sell: 3`

Or having just one pair for both exchanges in the config file with an explanation that they should be the same should suffice?

Reply:
Four will be better because it'll be clear that they are different in the case of crypto.com for ETH_BTC

Reply:
Ok, I will make that change.